themes/default/content/blog/mlops-the-ai-challenge-is-cloud-not-code/index.md

```diff
@@ -0,0 +1,189 @@
+
+The AI industry is stealing the show as tech's goldrush of the '20s. Just looking at ChatGPT's [record setting user growth](https://www.reuters.com/technology/chatgpt-sets-record-fastest-growing-user-base-analyst-note-2023-02-01/), and rapid 3rd party integration [by top brands](https://www.forbes.com/sites/bernardmarr/2023/05/30/10-amazing-real-world-examples-of-how-companies-are-using-chatgpt-in-2023/?sh=ed1c90f14418), it is not surprising the hype suggests this is the beginning of a major digital transformation.
+
+When it comes to taking advantage of AI/ML in your own organization, realizing the benefits comes with major obstacles, not least of which is the almost complete lack of well practiced platform engineering architectures, industry consensus, and operational knowledge. Which is why it is easy to say, *The most difficult challenge that we face with AI today is a problem of cloud orchestration*.
+
+This high-level diagram easily illustrates the complexity involved.
+
+> Figure 1. A sneak peak at the map of ML development and product path
+![A sneak peak at the map of ML development and product path](image.png)
+
+Intended to help users blaze a trail of success, this blog post is the first installment of the *Pulumi Python + MLOps* series. Through the journey of this series we will start from the beginning with a basic app and the simple cloud deployment architecture in this post, and follow the chatbot development through its evolution into a robust and resilient cloud native service.
+
+> # *The most difficult challenge that we face with AI today is a problem of cloud orchestration*
+
+Instructure as code (IaC) is a key component to solving the cloud orchestration challenge present in AI/ML. By applying the rigor and precision of software development practices to cloud operations, IaC offers AI/ML professionals an efficient, reliable, and predictable way to develop at the highest velocity.
+
+Before we get into the details of using Pulumi to deploy a private LlaMa 2 API chatbot service---in 30 minutes or less, of course---let's start with some quick terminology/context:
+
+- Katwalk Server is a demo AI application written to host your choice of LLM either locally or in the cloud, and serve it as an OpenAI API compatible service. As a practical, hands-on introduction to IaC for the MLOps space, this project aims to demystify the concepts and showcase the benefits Pulumi can bring to your AI/ML projects.
+- Large Language Models or LLMs (like ChatGPT) are machine learning models trained to generate human-like conversational text which have already become essential tools for businesses and individuals, transforming every day tasks and how we interact with technology.
+- [LlaMa](https://ai.meta.com/llama/) is an LLM created by Meta, available for free for research and commercial use.
+
+> Figure 2. Or just let ChatGPT describe itself
+![Alt text](image-1.png)
+
+## Deploy a private LlaMa 2 API chatbot service ... in 30 minutes or less!
```

Comment:
I'd consider hinting at this at the very beginning, along the lines of 'in 30m i'll give you working code, free pulumi access and then you'll be able to create your first chatbot and impress your friends, your boss, your team etc?; or some sort of hook?